### PR TITLE
raft-optical-flow

### PR DESCRIPTION
**Pull request by yangsu2022**
Hi @raymondlo84  @eaidova 
This is my new PR.
The old one: https://github.com/openvinotoolkit/openvino_notebooks/pull/966

### CODE REVIEW

File: notebooks/210-raft-optical-flow/210-raft-optical-flow.ipynb [hunk 1]
```diff
@@ -0,0 +1,700 @@
+{
```
Comment (eaidova) [inline notebooks/210-raft-optical-flow/210-raft-optical-flow.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.        if device != "CPU":
</pre><p>do you tested this branch?</p><p>Probably it is accidentally copied from another notebook (where dynamic input shape was replaced with static for inference on GPU device)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (yangsu2022) [reply notebooks/210-raft-optical-flow/210-raft-optical-flow.ipynb]: Thanks for the review, I delete it now

[the remark this answers is not in the document]
Comment (yangsu2022) [reply notebooks/210-raft-optical-flow/210-raft-optical-flow.ipynb]: understood, I remove this, no plan for quantization right now. After the new release, I could try it with nncf..

**Review by eaidova — APPROVED** [review]

**Comment by ryanloney** [comment]
I tried running on Arc A770 dGPU and this is where it fails: 
image

**Comment by yangsu2022** [comment]
> I tried running on Arc A770 dGPU and this is where it fails: image

Yes, OV22.3 not support 7D tensor on GPU, which fixed with OV23.0.
https://jira.devtools.intel.com/browse/CVS-100599?jql=text%20~%20%22raft%22

**Comment by ryanloney** [comment]
@yangsu2022 sorry, I was running in wrong env. I tried with pre-release 23.0 and it's running on GPU now. 

BTW why is benchmark_app run twice in the notebook? 

Here is what I'm seeing for Arc A770 dGPU benchmark: 
image

Unfortunately this is what the visualization looks like when running on iGPU and dGPU: 
image

image


I'm using `2023.0.0.dev20230427` from PyPI

**Pull request closed without merging**